website/addons/zotero/tests/test_models.py

```diff
@@ -34,337 +42,20 @@ def test_handle_callback(self):
-        assert_equal(api, mock_zotero())
-
-    @mock.patch('website.addons.zotero.model.Zotero')
-    def test_api_cached(self, mock_zotero):
-        # Repeated calls to .api returns the same object
-        self.node_settings._api = 'testapi'
-        api = self.node_settings.api
-        assert_false(mock_zotero.called)
-        assert_equal(api, 'testapi')
-
-    def test_set_auth(self):
-        external_account = ZoteroAccountFactory()
-        self.user.external_accounts.append(external_account)
-        self.user.save()
-
-        # this should be reset after the call
-        self.node_settings.zotero_list_id = 'anything'
-
-        self.node_settings.set_auth(
-            external_account=external_account,
-            user=self.user
-        )
-
-        # this instance is updated
-        assert_equal(
-            self.node_settings.external_account,
-            external_account
-        )
-        assert_equal(
-            self.node_settings.user_settings,
-            self.user_settings
-        )
-        assert_is_none(
-            self.node_settings.zotero_list_id
-        )
-
-        # user_settings was updated
-        # TODO: The call to grant_oauth_access in set_auth should be mocked
-        assert_true(
-            self.user_settings.verify_oauth_access(
-                node=self.node,
-                external_account=external_account,
-            )
-        )
-
-    def test_set_auth_wrong_user(self):
-        external_account = ZoteroAccountFactory()
-        self.user.external_accounts.append(external_account)
-        self.user.save()
-
-        with assert_raises(PermissionsError):
-            self.node_settings.set_auth(
-                external_account=external_account,
-                user=UserFactory()
-            )
-
-    def test_deauthorize(self):
-        self.node_settings.external_account = ZoteroAccountFactory()
-        self.node_settings.zotero_list_id = 'something'
-        self.node_settings.user_settings = self.user_settings
-        self.node_settings.save()
-
-        assert_true(self.node_settings.zotero_list_id)
-        self.node_settings.deauthorize(auth=Auth(self.user))
-        self.node_settings.save()
-        assert_is(self.node_settings.user_settings, None)
-        assert_is(self.node_settings.zotero_list_id, None)
-
-        last_log = self.node.logs[-1]
-        assert_equal(last_log.action, 'zotero_node_deauthorized')
-        params = last_log.params
-        assert_in('node', params)
-        assert_in('project', params)
-
-    def test_clear_auth(self):
-        self.node_settings.external_account = ZoteroAccountFactory()
-        self.node_settings.zotero_list_id = 'something'
-        self.node_settings.user_settings = self.user_settings
-        self.node_settings.save()
-
-        self.node_settings.clear_auth()
-
-        assert_is_none(self.node_settings.external_account)
-        assert_is_none(self.node_settings.zotero_list_id)
```

Comment:
[`test_clear_settings`](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/addons/base/testing/models.py#L226) secretly does the same thing, as [`AddonCitationsNodeSettings.folder_id`](https://github.com/CenterForOpenScience/osf.io/pull/4962/files#diff-fcda41ab988f395bf849bacc226f5b76R45) just returns `list_id`.